app/controllers/application_controller.rb

```diff
@@ -2279,7 +2279,7 @@ def find_by_id_filtered(db, id)
       raise msg
     end
 
-    Rbac.filtered_object(db_obj, :user => current_user) ||
+    Rbac.filtered_object(db_obj, :user => current_user, :named_scope => options[:rbac_scope]) ||
```

Comment:
Could we use `:named_scope` coming into this method as well?
Just throwing it out there so we can keep parameters consistent